academy/3-cosmos-hub/4-atom-staking.md

```diff
@@ -248,4 +248,4 @@ To summarize, this section has explored:
 
 <!--## Next up
 
-In the [next chapter](../2-cosmos-concepts/index.md), you will continue to explore the Cosmos SDK and learn about the underlying blockchain application architecture.-->
+In the [next chapter](../4-ibc/index.md), you will learn about the Inter-Blockchain Communication Protocol.-->
```

Comment:
Re-check once content is final and structure for portal deployment is final and change if needed.